html-tables/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <thead>
+      <tr>
+        <th>Student Name</th>
+        <th>Course</th>
+        <th>Grade</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th style="text-align:left">Grace Hopper</th>
```

Comment:
for the body, you want to use `td` elements. That will make it so they align left by default